### PR TITLE
Update docs and add test

### PR DESCRIPTION
**Pull request by matthewkotila**

### CODE REVIEW

File: qa/L0_quick_start/test.sh [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  --run-config-search-preferred-batch-size-disable true ; \
+  if [ $? -ne 0 ] ; then RET=1 ; fi
+
+mkdir analysis_results ; \
+  if [ $? -ne 0 ] ; then RET=1 ; fi
+
+model-analyzer analyze \
+  --analysis-models add_sub \
+  -e analysis_results ; \
+  if [ $? -ne 0 ] ; then RET=1 ; fi
```
Comment (debermudez) [inline qa/L0_quick_start/test.sh:31]: Lets have the return value accumulate. This way we know at a glance, how poorly the test performed.

**Review by debermudez — CHANGES_REQUESTED** [review]

**Review by tgerdesnv — CHANGES_REQUESTED** [review]

File: qa/L0_quick_start/test.sh (hunk 1 above)
Comment (matthewkotila) [reply qa/L0_quick_start/test.sh]: discussed offline--updated code to exit early in these cases

File: docs/install.md [hunk 2]
```diff
@@ -18,9 +18,38 @@ limitations under the License.
 
 There are 4 ways to use Triton Model Analyzer:
 
+## Building the Dockerfile
+
+The recommended way to use Model Analyzer is by building the Model Analyzer's docker yourself. First, clone the
```
[the remark this answers is not in the document]
Comment (matthewkotila) [reply docs/install.md]: swapped and reworded 👍

File: docs/launch_modes.md [hunk 3]
```diff
@@ -15,7 +15,7 @@ limitations under the License.
 -->
 # Launch Modes
 
-Triton Model Analyzer's `profile` subcommand supports three different launch
+Triton Model Analyzer's `profile` subcommand supports four different launch
```
[the remark this answers is not in the document]
Comment (matthewkotila) [reply docs/launch_modes.md:18]: mentioned 👍

File: qa/L0_quick_start/test.sh [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+RET=0
+
+model-analyzer profile \
+  -m /quick_start_repository/ \
```
[the remark this answers is not in the document]
Comment (matthewkotila) [reply qa/L0_quick_start/test.sh]: changed to valid path 👍

**Review by tgerdesnv — APPROVED** [review]

**Review by debermudez — APPROVED** [review]

**Pull request merged**